### PR TITLE
feat(storybook): Side Menu

### PR DESCRIPTION
**Pull request by FIZLIN**

### CODE REVIEW

File: packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import {ComponentMeta, Story} from '@storybook/react';
+
+import SideMenu, {SideMenuPropsInterface} from './SideMenu';
+
+export default {
+  title: 'Molecules/SideMenu',
+  component: SideMenu
+} as ComponentMeta<typeof SideMenu>;
+
+const Template: Story<SideMenuPropsInterface> = (args) => <SideMenu {...args} />;
```
Comment (PAKovalev) [inline packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.stories.tsx]: Make `useState` for storing active id.

File: packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  orientation?: SideMenuOrientation;
+}
+
+const SideMenu: FC<SideMenuPropsInterface> = ({
+  sideMenuItems,
+  activeIdx,
+  orientation = 'right',
+  onMenuItemSelection
+}) => {
+  const [activeMenuItemIdx, setActiveMenuItemIdx] = useState(activeIdx || 0);
```
Comment (PAKovalev) [inline packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.tsx]: Don't need this. Active id should come from props.

File: packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+          >
+            <Hexagon
+              type="primary"
+              isActive={idx === activeMenuItemIdx}
+              iconName={sideMenuItem.iconName}
+            />
+            <MenuLabel text={sideMenuItem.label} type={orientation} />
+            {sideMenuItem.pinNumber && (
+              <styled.MenuItemNumberPin className={cn(orientation === 'left' && 'inverted')}>
+                {sideMenuItem.pinNumber < 10 ? '0' : ''}
```
Comment (PAKovalev) [inline packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.tsx]: Make a const for `10` above.

File: packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+const SideMenu: FC<SideMenuPropsInterface> = ({
+  sideMenuItems,
+  activeIdx,
+  orientation = 'right',
+  onMenuItemSelection
+}) => {
+  const [activeMenuItemIdx, setActiveMenuItemIdx] = useState(activeIdx || 0);
+
+  const onClick = (idx: number) => {
```
Comment (PAKovalev) [inline packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.tsx]: Don't need this also.

**Review by PAKovalev — CHANGES_REQUESTED** [review]

File: packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.tsx (hunk 4 above)
Comment (FIZLIN) [reply packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.tsx]: ✅

File: packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.tsx (hunk 3 above)
Comment (FIZLIN) [reply packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.tsx]: ✅

File: packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.tsx (hunk 2 above)
Comment (FIZLIN) [reply packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.tsx]: ✅

File: packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.stories.tsx (hunk 1 above)
Comment (FIZLIN) [reply packages/ui-kit-storybook/src/molecules/SideMenu/SideMenu.stories.tsx]: ✅

**Review by PAKovalev — APPROVED** [review]

**Pull request merged**